views/widgets/hero-slider-default.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/* @var $image string */
+/* @var $instance array */
+?>
+
```

Comment:
please add example of printing before/after widget parts

Reply:
added examples